project/Deps.scala

```diff
@@ -22,9 +27,12 @@ object Deps {
 
   val boringssl = "io.netty" % "netty-tcnative-boringssl-static" % "2.0.19.Final"
 
-  def zkCandidate =
-    ("com.twitter.common.zookeeper" % "candidate" % "0.0.84")
-      .exclude("com.twitter.common", "util")
+  def apacheZookeeper = ("org.apache.zookeeper" % "zookeeper" % "3.5.0-alpha")
```

Comment:
I think if we depend on finalge-serversets, we should automatically get this as a transitive dependency.